test/JDBC/expected/TestPoint-vu-verify.out

```diff
@@ -0,0 +1,25 @@
+SELECT location FROM POINTGEOM_dt;
```

Comment:
Instead of creating this file, we can add all new tests into the existing TestSpatialPoint-vu-*. You can just change them from .sql to .txt

Reply:
Done